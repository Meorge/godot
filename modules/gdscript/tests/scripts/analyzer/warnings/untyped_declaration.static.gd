func no_exec_test():
    var my_var = 42
    print(my_var)

func test():
    pass
