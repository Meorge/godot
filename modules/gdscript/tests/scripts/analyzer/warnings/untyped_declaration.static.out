GDTEST_OK
~~ WARNING at line 1: (UNTYPED_DECLARATION) Function "no_exec_test()" has no static return type.
~~ WARNING at line 2: (UNTYPED_DECLARATION) Variable "my_var" has no static type.
~~ WARNING at line 5: (UNTYPED_DECLARATION) Function "test()" has no static return type.
